Makefile

```diff
@@ -38,7 +38,7 @@ report:
 	$(GO) tool cover -html=/tmp/cov.out -o ./report.html
 	rm /tmp/cov.out
 
-EXAMPLES := $(wildcard examples/*)
+EXAMPLES := $(filter-out $(wildcard examples/*.go), $(wildcard examples/*))
```

Comment:
```suggestion
EXAMPLES := $(dir $(wildcard examples/*/main.go))

examples: $(EXAMPLES)

.PHONY: $(EXAMPLES)
$(EXAMPLES):
    @echo Running example: $@
    $(GO) run ./$@
```

Reply:
I can't make a change suggestion for the entire thing because GitHub doesn't allow making suggestions on deleted or untouched lines, so you'd have to update the commit, remove the other lines and replace the spaces with tabs.